dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultDataSetMetadataExportService.java

```diff
@@ -159,6 +159,10 @@ public ObjectNode getDataSetMetadata() {
     List<OptionSet> optionSets = sortById(getOptionSets(dataElements));
 
     dataSetCategoryCombos.remove(defaultCategoryCombo);
```

Comment:
This is the reason for the bug - we're removing the default `catCombo` from dataSet. So if this is used as the `attribute` `catCombo`, but not in any `dataElement` - it's not included.